Sources/Benchmark/BenchmarkMetric.swift

```diff
@@ -186,9 +200,9 @@ public extension BenchmarkMetric {
 #if swift(>=5.8)
     @_documentation(visibility: internal)
 #endif
-extension BenchmarkMetric {
-    init(_ textualMetric: String) {
-        switch textualMetric {
+public extension BenchmarkMetric {
+    init?(argument: String) {
```

Comment:
Why fallible? It seems object will be always created?

Reply:
It is to conform to ExpressibleByArgument protocol from SAP.

Reply:
https://apple.github.io/swift-argument-parser/documentation/argumentparser/expressiblebyargument/